sys/cheri/cheri.h

```diff
@@ -107,6 +107,14 @@ extern void * __capability swap_restore_cap;
 #ifdef __CHERI_PURE_CAPABILITY__
 /* Root kernel capability */
 extern void * __capability kernel_root_cap;
+#ifdef __aarch64__
```

Comment:
Maybe `#ifdef VMM` and rename vmm_el2_root_cap to vmm_hyp_root_cap?

Reply:
There is no VMM kernel option. vmm is never compiled into the kernel, it's always loaded as a kernel module.

The typical pattern would be to add a VMM_SUPPORT kernel option and make this conditional on that, but absent other code which could use it that seems excessive.

Reply:
Hm. The option does exist (files.arm64 is modified, and options.arm64 has VMM opt_global.h) but it's true this wouldn't work with the normal module approach.